### DISCUSSION TITLE
[FEATURE] 能在配置文件中添加字体字段吗？

### DISCUSSION THREAD

**Discussion by userxiaosi** [hugo-fixit/FixIt#159] [Q&A] [answered]
### Describe the feature you want 描述你的功能需求

能在配置文件中添加字体字段吗？默认字体I与L辨识度不高，希望能在配置文件中增加相关选项

**Comment by userxiaosi** [comment]
纠正，是大写i与小写L辨识度不高

**Comment by Lruihao** [comment, marked as answer]
@userxiaosi 你可以将 [_custom.scss](https://github.com/Lruihao/FixIt/blob/master/assets/css/_custom.scss) 复制到自己博客相似相同目录，在 _custom.scss 里你可以自定义任何的 CSS 样式，比如全局修改字体你可以这样写

```css
html {
  font-family: 'system-ui, -apple-system'
}
```

或者 [_override.scss](https://github.com/Lruihao/FixIt/blob/master/assets/css/_override.scss) 覆盖全局的 scss 字体变量  [`$global-font-family`](https://github.com/Lruihao/FixIt/blob/master/assets/css/_variables.scss#L10)

更多请查看 https://fixit.lruihao.cn/zh-cn/theme-documentation-basics/#33-自定义样式

- 相似问题 https://github.com/Lruihao/FixIt/discussions/80

**Comment by userxiaosi** [comment]
非常感谢
